iwp_website/controllers/subscription_request.py

```diff
@@ -102,6 +96,7 @@ def prepare_subscription_request_value(self, struct, qcontext=None):
             iban = partner.bank_ids[0].acc_number
         except IndexError:
             _logger.error('no account set for partner %s' % partner)
+            raise ValueError(_('no account set for partner %s' % partner))
```

Comment:
exposing a stacktrace to the website isn't a good idea. A best option would be to it display in an error message